### ISSUE TITLE
Add owners

### ISSUE THREAD

**Issue by RobLoach** [jstransformers/jstransformer-riot#1]
http://npm.im/jstransformer-riotjs

**Comment by tunnckoCore** [comment]
It's to different things, read in riotjs transformer repo.
We can delete this repo one cuz i dont have time for it.

About other issues for "add owners", sorry, dont have time and enviroment atm - i hope mid december.

**Comment by RobLoach** [comment]
@tunnckoCore Can you than just run `npm unpublish jstransformer-riotjs` so that there aren't any conflicts?

**Comment by tunnckoCore** [comment]
No, I can't. Don't have stable machine currently (upgrading components, config arch and etc in lil' free time) and time.
